debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerSnapshotChangeEventSource.java

```diff
@@ -193,8 +199,13 @@ protected void complete(SnapshotContext snapshotContext) {
     }
 
     @Override
-    protected String getSnapshotSelect(SnapshotContext snapshotContext, TableId tableId) {
-        return String.format("SELECT * FROM [%s].[%s]", tableId.schema(), tableId.table());
+    protected Optional<String> getSnapshotSelect(SnapshotContext snapshotContext, TableId tableId) {
+        if (snapshotOverrides.containsKey(tableId)) {
```

Comment:
This should also go up to base class. The base class should check if there are overrides and if yes then use them. If not then this method should be called.

Reply:
Done